Allow helpers to logblock lookup

Legoman99573 opened the pull request:
This makes it easier for grief reports.

Legoman99573 committed:
Allow helpers to logblock lookup

This makes it easier for grief reports.

sweepies commented on plugins/PermissionsEx/permissions.yml:
```diff
@@ -113,6 +113,7 @@ groups:
     - permissions.user.promote.default
     - permissions.user.rank.demote
     - permissions.user.rank.promote
+    - logblock.lookup
```
Add `logblock.tools.tool` and `logblock.tools.toolblock`

From the wiki:
- logblock.lookup
> Grants user access to /lb lookup (Users can't use their tools without this node!)

Legoman99573 replied:
sorry
will resubmit it

sweepies replied:
You can push more commits to the branch and they'll automatically be reflected here.

Legoman99573 replied:
pushed it ;)

Legoman99573 committed:
Corrected perms

@sweepyoface did mention i forgot something. I haven't used logblock since 1.8, so I forgot to add some perms needed or my pull would be useless.

ScruffyRules commented:
Did this server side, please close.

Legoman99573 commented:
Closed upon request <3
